tests/test_edgeql_expr_aliases.py

```diff
@@ -1304,6 +1304,446 @@ async def test_edgeql_aliases_schema_types_03(self):
             []
         )
 
+    async def _check_ddl_type_changes(self, ddls_and_expected_types):
+        """Check newly created schema types after a series of ddl stmts."""
+        existing_types = await self.con.query("select schema::Type.id")
+
+        for ddl, expected_types in ddls_and_expected_types:
+            await self.con.execute(f'''
+                {ddl};
+            ''')
+
+            await self.assert_query_result(
+                f'''
+                    select schema::Type {{
+                        name,
+                        from_alias,
+                        type_name := (.__type__.name),
+                    }}
+                    filter .id not in {{{','.join(
+                        f'<uuid>"{str(existing_type)}"'
+                        for existing_type in existing_types
+                    )}}}
```

Comment:
Can you do this by passing in a parameter instead? It doesn't really matter but is cleaner